### PR TITLE
8.1-3-Pass-2 for Review

### PR DESCRIPTION
**Pull request by CJStegall**
TC8.1-3

### CODE REVIEW

File: TC8.1-3-pass-2 [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+|IC| All visible text AND images of text (except those noted in Limitations, Assumptions, or Exceptions above)|
+| 8.1-2| If the contrast ratio is less than 4.5:1, check that the ratio is at least 3:1 AND the font meets one of the following criteria: [SC 1.4.3]| The contrast is at least 14 point (18.62 pixels) AND bold (at least 700 font weight)|
```
Comment (kengdoj) [inline TC8.1-3-pass-2:51]: Please add the first test instruction and the contrast found in the Result column
8.1-1 | Determine the contrast ratio of foreground text and background. | (enter contrast ratio found |

File: testcases/TC8.1-2-fail-4.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+|IC| All visible text AND images of text (except those noted in Limitations, Assumptions, or Exceptions above)|
+| 8.1-1| Determine the contrast ratio of foreground and background in the image of text. | The contrast ratio for the text is 4.5:1 |
```
Comment (kengdoj) [inline testcases/TC8.1-2-fail-4.md:51]: the last column for the result should explain the actual contrast ratio.

File: testcases/TC8.1-2-fail-4.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+|IC| All visible text AND images of text (except those noted in Limitations, Assumptions, or Exceptions above)|
+| 8.1-1| Determine the contrast ratio of foreground and background in the image of text. | The contrast ratio for the text is 4.5:1 | 
+| 8.1-2| Check that the contrast ratio is at least 4.5:1. [SC 3.14] | The contrast ratio is more than 4.5:1 (it is 2.6:1) |
```
Comment (kengdoj) [inline testcases/TC8.1-2-fail-4.md:52]: if the contrast ratio is 2.6:1, it is less.

Comment (kengdoj) [reply testcases/TC8.1-2-fail-4.md:52]: Also please correct the SC number.

File: testcases/TC8.1-2-fail-4.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+| Results | Both of the above checks fail, then Baseline Test 8.1-Contrast Minimum.  | Fail |
```
Comment (kengdoj) [inline testcases/TC8.1-2-fail-4.md:53]: Copy the result from the Baseline Test Results for the second column: 
| | If both of the above checks fail, then Baseline Test 8.1-ContrastMinimum fails | Fail: then your explanation |

File: testcases/TC8.1-2-fail-4.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+# Brief description of the rationale for the expected result
+result-descr: The foreground and background text and size contrast ratio in the code sample provide sufficient contrast in an image of text.
+
+# URL for the code sample
+# In the sample code file, add id="tc_code" to the 
+# element that contains the relevant code snippet.
+#
+# Then upload the code sample to the 'testfiles' folder 
+# and provide the link (and only the url) below.
+sample: /testfiles/TF08/08.1-2-fail-4.html
```
Comment (kengdoj) [inline testcases/TC8.1-2-fail-4.md:41]: Is this file in a different PR?

**Review by kengdoj — CHANGES_REQUESTED** [review]
Thanks, Cecilia! Please see comments.

File: testcases/TC8.1-2-fail-4.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+# Baseline test
+descr: Detect the foreground and background text and size contrast ratio. Determine whether contrast ratio is sufficient. The text in the code sample is sufficient contrast between the foreground and background.
+
+
+# Reference and link to the applicable ICT Baseline test
+app-basline:|[8.1 Test Procedure for Contrast Minimum]https://section508coordinators.github.io/ICTTestingBaseline/08Contrast.html#81-test-procedure-for-contrast-minimum
+
+  **Baseline Test ID:** 8.1-Contrast
+    
+  **Test Instruction:** 1
```
Comment (kengdoj) [inline testcases/TC8.1-2-fail-4.md:25]: the filename indicates this would be for instruction 2. Can you double check?

**Review by kengdoj — CHANGES_REQUESTED** [review]

File: TC8.1-3-pass-2 [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+# Description of the Test Case, the included code sample, test considerations,
+# and rationale for the expected result according to the applicable ICT
+
+# Baseline test
+descr: Detect the foreground and background text and size contrast ratio. Determine whether contrast ratio is sufficient. The text in the code sample is sufficient contrast between the foreground and background.
+
+
+# Reference and link to the applicable ICT Baseline test
+app-basline:|[8.1 Test Procedure for Contrast Minimum]https://section508coordinators.github.io/ICTTestingBaseline/08Contrast.html#81-test-procedure-for-contrast-minimum
```
[the remark this answers is not in the document]
Comment (CJStegall) [reply TC8.1-3-pass-2:21]: Nope the typo is not in the original template.

[the remark this answers is not in the document]
Comment (CJStegall) [reply TC8.1-3-pass-2:21]: Done

File: TC8.1-3-pass-2 [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+sample: /testfiles/TF08/08.1-3-pass-2.html
+
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+|IC| All visible text AND images of text (except those noted in Limitations, Assumptions, or Exceptions above)|
```
[the remark this answers is not in the document]
Comment (CJStegall) [reply TC8.1-3-pass-2:50]: Done

[the remark this answers is not in the document]
Comment (kengdoj) [reply TC8.1-3-pass-2:49]: I don't see "visible text found" in the last column for identify content.

File: TC8.1-3-pass-2 [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+# Baseline test
+descr: Detect the foreground and background text and size contrast ratio. Determine whether contrast ratio is sufficient. The text in the code sample is sufficient contrast between the foreground and background.
+
+
+# Reference and link to the applicable ICT Baseline test
+app-baseline:|[8.1 Test Procedure for Contrast Minimum](https://section508coordinators.github.io/ICTTestingBaseline/08Contrast.html#81-test-procedure-for-contrast-minimum)
+
+  **Baseline Test ID:** 8.1-Contrast
+    
+  **Test Instruction:** 1
```
Comment (kengdoj) [inline TC8.1-3-pass-2:24]: the filename has 8.1-3. Please check the instruction number?

File: testfiles/TF08/08.1-3-pass-2.html [hunk 10]
```diff
@@ -0,0 +1,10 @@
+
+ <!DOCTYPE html>
+<html>
+<head>
+<title>HTMLFont size</title>
+</head>
+<body>
```
[the remark this answers is not in the document]
Comment (kengdoj) [reply testfiles/TF08/08.1-3-pass-2.html:7]: I don't see this change?

File: testcases/TC8.1-2-fail-4.md (hunk 5 above)
Comment (kengdoj) [reply testcases/TC8.1-2-fail-4.md:38]: I can't approve this one without the test file.

**Review by kengdoj — DISMISSED** [review]
please see comments.

**Review by drewnielson — APPROVED** [review]
Approved with edits

**Pull request merged**